src/prognostic_equations/implicit/manual_sparse_jacobian.jl

```diff
@@ -543,8 +543,26 @@ function update_jacobian!(alg::ManualSparseJacobian, cache, Y, p, dtγ, t)
     end
 
     if use_derivative(diffusion_flag)
+        turbconv_params = CAP.turbconv_params(params)
+        FT = eltype(params)
+        (; vert_diff) = p.atmos
+        (; ᶜp) = p.precomputed
+        if vert_diff isa DecayWithHeightDiffusion
+            ᶜK_h = compute_eddy_diffusivity_coefficient(Y.c.ρ, vert_diff)
+            ᶜK_u = ᶜK_h
+        elseif vert_diff isa VerticalDiffusion
+            ᶜK_h = compute_eddy_diffusivity_coefficient(Y.c.uₕ, ᶜp, vert_diff)
+            ᶜK_u = ᶜK_h
+        else
+            (; ᶜtke⁰,) = p.precomputed
+            ᶜmixing_length = p.scratch.ᶜtemp_scalar_6
```

Comment:
@dennisYatunin Is there some way to encapsulate this recurring materialize in scratch pattern, so it's easy to remove later?